puppet-profiles/springboot/.devcontainer/README.md

```diff
@@ -0,0 +1,38 @@
+# devcontainer
```

Comment:
What is that file for?

Reply:
File created by the PDK convert command, no idea what's used for except making the Puppet module compliant... ;-)